e2e/helpers/puppeter_helper.js

```diff
@@ -80,4 +83,15 @@ module.exports = class PuppeteerHelpers extends Helper {
       return undefined;
     }
   }
+
+  async runAccessibilityTest() {
+
+    if (!testConfig.TestForAccessibility) {
+      return;
+    }
+    const url = await this.helpers['Puppeteer'].grabCurrentUrl();
+    const {page} = await this.helpers['Puppeteer'];
+
+    runAccessibility(url, page);
```

Comment:
should it be:
wait puppeter_helper.js

Reply:
you prob want to `await` the result runAccessibility(...)